e2e/testing/chain_ops.go

```diff
@@ -14,8 +14,8 @@ func (chain *TestChain) ExecuteGovProposal(proposerAcc Account, expPass bool, pr
 
 	// Get params
 	k := chain.app.GovKeeper
-	depositCoin := k.GetDepositParams(chain.GetContext()).MinDeposit
-	votingDur := k.GetVotingParams(chain.GetContext()).VotingPeriod
+	depositCoin := k.GetParams(chain.GetContext()).MinDeposit
```

Comment:
could we get params once? Ik it's a test and won't affect performance much but just for the sake of being slightly cleaner

Reply:
👍🏻